include/deal.II/base/vectorization.h

```diff
@@ -544,12 +580,46 @@ class VectorizedArray<double>
    * 64 bytes, as opposed to casting a double address to
    * VectorizedArray<double>*.
    */
-  void store (double *ptr) const
+  DEAL_II_ALWAYS_INLINE void store (double *ptr) const
   {
     _mm512_storeu_pd (ptr, data);
   }
 
   /**
+   * Gather operation.
+   */
+  DEAL_II_ALWAYS_INLINE void gather (const double *base_ptr,
```

Comment:
If you don't mind too much, could you break the line so that the inline part is on a separate line and the type in column 1? I think that would make it easier to read.